### PR TITLE
implemented methods

### PR DESCRIPTION
**Pull request by ivanlavrinenko**
login method in AuthentificationService and findByEmail in UserService are implemented

### CODE REVIEW

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 1]
```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
```
Comment (kerrrusha) [inline src/main/java/mate/academy/service/AuthenticationService.java:16]: should be class-level `private final` field

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 2]
```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
...
+        User user = userService.findByEmail(email);
+        if (user == null) {
+            return false;
+        }
+        return user.getPassword().equals(password);
```
Comment (kerrrusha) [inline src/main/java/mate/academy/service/AuthenticationService.java:21]: it's ok, but as an extra challenge, you can try to rewrite it in 1 line) if you want, think about how to do it

**Review by kerrrusha — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/service/AuthenticationService.java (hunk 1 above)
Comment (ivanlavrinenko) [reply src/main/java/mate/academy/service/AuthenticationService.java]: I have just added new commit on by branch.

Comment (ivanlavrinenko) [reply src/main/java/mate/academy/service/AuthenticationService.java]: > should be class-level `private final` field

I have just added new commit on by branch, but can't see if it's checked by Java CI.

**Review by Rommelua — APPROVED** [review]

**Pull request closed without merging**